### ISSUE TITLE
domain_exporter return i/o timeout error for some gTLD 

### ISSUE THREAD

**Issue by h1kkan** [caarlos0/domain_exporter#7]
Hi @caarlos0, i've faced an issue with some gTLD zones. 

When i try to check this zones:
```
.xyz
.space
.site
.online
.website
.tech
.online
.pw
```
they all failed with `i/o timeout` error, like this one `dial tcp 119.252.181.42:43: i/o timeout`

Also i saw that the f.e. `evenstranger.pw` at domainr (`https://domainr.com/evenstranger.pw?q=evenstranger.pw`) won't show us domain expiration date (or any whois data Oo), but `whois` command return expiration date:
```
whois evenstranger.pw | grep "Registry Expiry Date"
Registry Expiry Date: 2018-10-23T23:59:59.0Z
```

**Comment by caarlos0** [comment]
That's weird, I have a `.pw` domain, for example, and it works as expected:

```
λ curl 'localhost:9222/probe?target=watchub.pw'
# HELP domain_expiry_days time in days until the domain expires
# TYPE domain_expiry_days gauge
domain_expiry_days 179
# HELP probe_duration_seconds Returns how long the probe took to complete in seconds
# TYPE probe_duration_seconds gauge
probe_duration_seconds 0.773419824
```

**Comment by h1kkan** [comment]
I've figured out root cause of that issue - we're was blocked by whois service (not by domainr), tried to use single exporter with 1500+ domains :) 

But some zones return error `couldnt parse whois`, f.e. .LT zone: 
```
curl -g 'localhost:9222/probe?target=domreg.lt'
couldnt parse whois for domain: domreg.lt
```
```
whois domreg.lt | grep Exp
Expires:		2018-10-17
```

I'm a bit little occupied now, i'll prepare complete report by the end of the week.

**Comment by caarlos0** [comment]
this `domreg.lt` we should fix for sure.

the others I think you already figured out :D 

thanks

**Comment by caarlos0** [comment]
/remind me this issue in 8 hours

**Comment by caarlos0** [comment]
should be fixed on v1.1.0 :) 

thank
